app/models/tagset_models/tag_nomination.rb

```diff
@@ -68,19 +68,17 @@ def set_tag_status
 
   before_save :set_parented, unless: :blank_tagname?
   def set_parented
-    if type == "FreeformNomination"
-      # skip freeforms
-      self.parented = true
-    elsif (tag = Tag.find_by_name(tagname)) &&
-      ((!tag.parents.empty? && get_parent_tagname.blank?) || tag.parents.collect(&:name).include?(get_parent_tagname))
-      # if this is an existing tag and has matching parents, or no parent specified and it already has one
-      self.parented = true
-      self.parent_tagname ||= get_parent_tagname
-    else
+    self.parent_tagname ||= get_parent_tagname
```

Comment:
@Bilka2 I've pulled this to the beginning to address the bug @sarken found, plus your earlier comment. Did you want the references to `self.parent_tagname` below switched back too?

Reply:
Yes